web/rest/client/features/my/dashboard/getDashboard.feature

```diff
@@ -45,7 +45,7 @@ Feature: Retrieve dashboard
           "extensionNum": 4,
           "ddiNum": 1,
           "residentialDeviceNum": null,
-          "voiceMailNum": null,
+          "voiceMailMessageNum": null,
```

Comment:
This is a breaking change. Keep former name or at least document it in API changelog